readme.md

```diff
@@ -687,18 +687,18 @@ A function that returns an object representing Got options pointing to the next
 ###### \_pagination.filter
 
 Type: `Function`\
-Default: `(item, allItems) => true`
+Default: `(item, allItems, currentItems) => true`
 
-Checks whether the item should be emitted or not.
+Checks whether the item should be emitted or not. `currentItems` is an array of the emitted of items of the current response and `allItems` is an error of _all_ the previously emitted items.
 
 ###### \_pagination.shouldContinue
 
 Type: `Function`\
-Default: `(item, allItems) => true`
+Default: `(item, allItems, currentItems) => true`
 
-Checks whether the pagination should continue.
+Checks whether the pagination should continue. `currentItems` is an array of the emitted of items of the current response and `allItems` is an error of _all_ the previously emitted items.
```

Comment:
Error? That's incorrect.

```suggestion
Checks whether the pagination should continue. `allItems` is an array of the all emitted items, while `currentItems` is an array of the current response items.
```

Reply:
Of course, it was just a typo. Good catch 😊